PowerUp/app/src/main/res/layout/gamemap.xml

```diff
@@ -64,4 +123,12 @@
         android:layout_alignParentTop="true"
         android:alpha="@dimen/map_alpha" />
 
+    <Button
+        android:layout_width="50dp"
```

Comment:
provide an id to the given button and try to not hard-code the dimensions but strore them in dimens.xml file